wagtailio/project_styleguide/templates/patterns/atoms/svgs/arrow.html

```diff
@@ -0,0 +1,3 @@
+<svg class="{% if classes %}{{ classes }}{% endif %}" viewBox="0 0 16 11" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
```

Comment:
nitpick: `patterns/atoms/svgs/` -> `patterns/atoms/icons/`
especially when you've got `static/sass/atoms/icons.scss` ;)

Reply:
Good point, I've pulled this out from your further point anyway.